### ISSUE TITLE
Breakdown `marketing` slack channel

### ISSUE THREAD

**Issue by dlxclem** [dlxmedia/community#12]
The idea is to breakdown the marketing channel on Slack into multiple "topic based" channels so it's easier to follow, easier to post and keep the team updated, etc. It worked with the dev team when we created new channels per "project" and it's working already for the `prog-streamer-content` channel. 

I don't have the list of channels we should create though but here are some ideas:
- proj-eplay-blog
- proj-eplay-social
- proj-eplay-community (or eplay-community) for the new github community + the streamer community we are part of?

Please share what you think would be good, this list is purely a first shot at figuring this out together and in no way the final list of channels we should create :)

**Comment by Korrieanne** [comment]
I like the idea and agree with the above names, except having the 3rd one be eplay-community

**Comment by dlxclem** [comment]
@Korrieanne the idea is to have a channel for the new eplay community (on github) - what would you call it?

**Comment by Korrieanne** [comment]
Sorry, I was not clear, I would call it eplay-community vs proj-eplay-community

**Comment by ePlayEusebiu** [comment]
I agree with Korrie to have eplay-community (instead of proj-eplay-community) since it will be a public one and I will add proj-eplay-summit too

**Comment by MaeganePlay** [comment]
I like this idea, and I think having seperate channels for the blog and social media will be beneficial! There may be a need for other channels such as proj-eplay-assets (just an example) in the future, after we decide on our marketing strategy and cadence for 2023.

**Comment by dlxclem** [comment]
alright, how about:

`eplay-community`
`eplay-social`
`eplay-blog`

**Comment by ePlayEusebiu** [comment]
Looks good to me but I would include also **eplay-assets** (for store) and **eplay-tradeshows**

**Comment by Korrieanne** [comment]
Yes, I like them all and agree with the additions

**Comment by paulinaeplay** [comment]
**eplay-releases** or **eplay-go-to-market** also! ^
